radicle-node/src/service/reactor.rs

```diff
@@ -81,11 +81,6 @@ pub struct Reactor {
 }
 
 impl Reactor {
-    /// Emit an event.
-    pub fn event(&mut self, event: Event) {
-        self.io.push_back(Io::Event(event));
-    }
-
```

Comment:
You can also remove the `Io::Event` variant.

Reply:
[removed](https://github.com/radicle-dev/heartwood/compare/096b533df2b3e727e32ed084c3f5e301bafd5a6d..350133ebabb47b1ed79128eb691c771235b1253f#diff-d8b3517be454cad4430e080cbba053e9c346237f8f5ea1e4e6142a9e70ec62c8L25-L26)